simapp/simd/cmd/testnet.go

```diff
@@ -319,7 +321,7 @@ func collectGenFiles(
 			return err
 		}
 
-		nodeAppState, err := genutil.GenAppStateFromConfig(cdc, config, initCfg, *genDoc, genBalIterator)
+		nodeAppState, err := genutil.GenAppStateFromConfig(cdc, txEncodingConfig, config, initCfg, *genDoc, genBalIterator)
```

Comment:
`config` makes it bit odd for readability as there are multiple config arguments. can we use `tmConfig` or something?
```suggestion
		nodeAppState, err := genutil.GenAppStateFromConfig(cdc, txEncodingConfig, tmConfig, initCfg, *genDoc, genBalIterator)
```